spec/acceptance/lastEvaluatedKeyWithFilterAndPagingTest.spec.ts

```diff
@@ -188,38 +194,35 @@ describe('lastEvaluatedKeyWithFilterAndPagingTest', function ()
 
         let page1 = await nodenamo.list().from(User).filter(filter).order(false).limit(2).execute<User>();
         
-        assert.deepEqual(page1.items, [
-            user5,
-            user4
-        ]);
+        assert.deepEqual(page1.items[0], user5);
+        assert.deepEqual(page1.items[1], user4);
+        assert.equal(page1.items.length, 2);
         assert.isDefined(page1.firstEvaluatedKey)
         assert.isDefined(page1.lastEvaluatedKey)
 
         let page2 = await nodenamo.list().from(User).filter(filter).limit(2).order(false).resume(page1.lastEvaluatedKey).execute<User>();
 
-        assert.deepEqual(page2.items, [
-            user3,
-            user1
-        ]);
+        assert.deepEqual(page2.items[0], user3)
+        assert.deepEqual(page2.items[1], user1)
+        assert.equal(page2.items.length, 2);
         assert.isDefined(page2.firstEvaluatedKey);
         assert.isUndefined(page2.lastEvaluatedKey);
 
         let page1Again = await nodenamo.list().from(User).filter(filter).limit(2).order(true).resume(page2.firstEvaluatedKey).execute<User>();
 
-        assert.deepEqual(page1Again.items, [
-            user4,
-            user5
-        ]);
+        assert.deepEqual(page1Again.items[0], user4)
+        assert.deepEqual(page1Again.items[1], user5)
+        assert.equal(page1Again.items.length, 2);
         assert.isDefined(page1Again.firstEvaluatedKey)
         assert.isUndefined(page1Again.lastEvaluatedKey)
 
         let page2Again = await nodenamo.list().from(User).filter(filter).limit(2).order(false).resume(page1Again.firstEvaluatedKey).execute<User>();
 
-        assert.deepEqual(page2Again.items, [
-            user3,
-            user1
-        ]);
+        assert.deepEqual(page2Again.items[0], user3);
+        assert.deepEqual(page2Again.items[1], user1);
+        assert.equal(page2Again.items.length, 2);
         assert.isDefined(page2Again.firstEvaluatedKey);
+        assert.equal(page2.firstEvaluatedKey, page2Again.firstEvaluatedKey)
         assert.isUndefined(page2Again.lastEvaluatedKey);
```

Comment:
Can we add assert.deepEqual(page2Again.lastEvaluatedKey, page2.lastEvaluatedKey) ?